### PR TITLE
29 empty todo

### LINKED ISSUE #29: Can create an empty todo
This is a bug where you can create an empty todo. 

Expected behavior is that you should not be able to create an empty todo

### PR DESCRIPTION
**Pull request by jonathan-lindqvist**
The user can no longer create an empty todo, and the user now gets feedback if they try it.

Create a new scss file for classes that are added temporarily. 

Close #29

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,6 +16,7 @@
                 <div class="todo-list-card">
                     <h2 class="todo-title">Todo List</h2>
                     <div id="items"></div>
+                    <p id="todoErrorText">Can't create empty todo!</p>
```
Comment (MSaurus) [inline index.html:19]: "an empty todo" sounds better in my opinion

**Review by MSaurus — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -6,6 +6,7 @@
     <title>New-tab</title>
 
     <link rel="stylesheet" href="css/index.css">
```
[the remark this answers is not in the document]
Comment (jonathan-lindqvist) [reply index.html:8]: nice catch, didnt notice that, but yes we should be consistent!

File: index.html (hunk 1 above)
Comment (jonathan-lindqvist) [reply index.html]: Done

**Review by MSaurus — APPROVED** [review]
👍

**Pull request merged**